tests/mapping/test_mapping_script.py

```diff
@@ -0,0 +1,121 @@
+import unittest
+import rdflib
+import datetime
+import os
+import yaml
+from pathlib import Path
+
+from lebedigital.mapping.emodul_mapping import generate_knowledge_graph, get_date_time_value
+
+class TestMappingScript(unittest.TestCase):
```

Comment:
mapping is done for different experiments, in that case Youngs modulus test (which is called emodul_mapping). Thus, I would suggest to have a file in test/mapping that is called test_emodul_mapping.py). I would also suggest we stick to a single testing framework  (as Erik has already pointed out) and use pytest instead of unittest.